docs/source/algorithms/MuonMaxent-v1.rst

```diff
@@ -0,0 +1,64 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm calculates a single frequency spectrum from the time domain spectra recorded by multiple groups/detectors. 
+
+The time domian data :math:`D_k(t)`, where :math:`t` is time and :math:`k` is the spectrum number, has associated errors :math:`E_k(t)`. If the number of points chosen is greater than the number of time domain sata points then extra points are
+added with infinite errors. The time domain data prior to :code:`FirstGoodTime` also have their errors set to infinity. The algorithm will produce the frequency spectra :math:`f(\omega)` and this is assumed to be real and positive. 
+The upper limit of the frequency spectra is determined by :code:`MaxField`. The maximum frequency, :math:`\omega_\mathrm{max}` can be less than the Nyquist limit :math:`\frac{\pi}{\delta T}` if the instrumental frequency response function for 
+:math:`\omega>\omega_\mathrm{max}` is approximatley zero. The inital estimate of the frequency spectrum is flat. 
+
+The algorithm calculates an estimate of each time domain spectra, :math:`g_k(t)` by the equation
+
+.. math::  g_k(t)=(1+A_k \Re(\mathrm{IFFT}(f(\omega) R(\omega))\exp(-j\phi_k) ) ),
+
+where :math:`\Re(z)` is the real part of :math:`z`, :math:`\mathrm{IFFT}` is the inverse fast Fourier transform, :math:`\phi_k` is the phase and :math:`A_k` is the asymmetry of the of the  :math:`k^\mathrm{th}` spectrum. 
+The asymmetry is normalised such that :math:`\sum_k A_k = 1`. 
+The instrumental frequency response function, :math:`R(\omega)`, is  is in general complex (due to a 
+non-symmetric pulse shape) and is the same for all spectra. The values of the phases and asymmetries are fitted in the outer loop of the algorithm. 
+
+The :math:`\chi^2` value is calculated via the equation
+
+.. math:: \chi^2 = F\frac{\sum_{k,t} (D_k(t)-g_k(t))^2 }{E_k(t)^2},
+
+where :math:`F` is the :code:`Factor` and is of order 1.0 (but can be adjusted within the algorithm for a better fit). 
```

Comment:
By "(but can be adjusted within the algorithm for a better fit)" do you mean that the algorithm adjust F during run time or that the user can select what this value should be?

Reply:
ah, sorry. The F value can be changed by the user (run a new algorithm)